commandline/src/main/java/tech/pegasys/web3signer/commandline/Web3SignerBaseCommand.java

```diff
@@ -314,8 +320,11 @@ public boolean keystoreParallelProcessingEnabled() {
   }
 
   @Override
-  public int getWorkerPoolSize() {
-    return workerPoolSize;
+  public int getVertxWorkerPoolSize() {
+    if (deprecatedWorkerPoolSize != null) {
```

Comment:
What if both options are specified? I think that should be an error in that case